### ISSUE TITLE
Could be interesting to mention the "tidyverse"

### ISSUE THREAD

**Issue by InFlamUOB** [swcarpentry/r-novice-gapminder#719]
Given the chapters on ggplot, dplyr, tidyr... - it could be nice to mention the whole tidyverse universe too - maybe by including just a link in the seeking help chapter e.g "https://www.tidyverse.org" or before teaching ggplot or after tidyr. Knowing that there is a whole universe of as useful packages can be very nice to see and also is a good resource to learn more about those chapters.

**Comment by naupaka** [comment]
I agree -- perhaps this is best done as a callout box just as a reminder for the instructor to mention it.

**Comment by crschul** [comment]
At a workshop I helped with we had a couple of cheatsheets floating around and provided participants with this link to print their own: https://www.rstudio.com/resources/cheatsheets/

**Comment by jcoliver** [comment]
@crschul Indeed. A link to the RStudio page of cheat sheets would be a welcome addition to the ggplot lesson.

**Comment by mantoniou** [comment]
I believe it would also be important to mention that tidyverse packages are **fully compatible with each other**, helping the user to easily combine commands from the different packages (e.g. dplyr, tidyr, ggplot) into a single statement by using the pipe command **%>%**

**Comment by matthieu-bruneaux** [comment]
Thank you for your suggestions @crschul and @mantoniou!

Would anyone like to submit one or two PRs to:

- Add a link to the RStudio page of cheatsheets (for example in the ggplot lesson).
- Update the "Tip: Tidyverse" box in `_episodes_rmd/13-dplyr.Rmd` to mention that the tidyverse packages are designed to work gracefully together and with the pipe?

**Comment by skanwal** [comment]
> Update the "Tip: Tidyverse" box in _episodes_rmd/13-dplyr.Rmd to mention that the tidyverse packages are designed to work gracefully together and with the pipe?

This has been attempted in https://github.com/swcarpentry/r-novice-gapminder/pull/823#pullrequestreview-1376681325

> Add a link to the RStudio page of cheatsheets (for example in the ggplot lesson).
This needs to be still tackled.

**Comment by naupaka** [comment]
Thanks @Phylloxera -- it's in the `episodes` directory in the repo:
https://github.com/swcarpentry/r-novice-gapminder/blob/main/episodes/08-plot-ggplot2.Rmd

**Comment by Phylloxera** [comment]
Added a link to the RStudio page of cheatsheets in the ggplot lesson. #830
